style-guide/svg/edit.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+        "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Do we really need the xml prolog and the DTD? Asking because I relly don't know, but seems weird to me.